Ramone/service providers (#96)
HorizenOfficial/evm-documentation: docs/horizen_eon/06-service_providers/eon_service_providers.md

---
title: EON-compatible Service Providers
---

> The EON ecosystem is perpetually changing with new partners and integrations. To accommodate the compatibility of EON platform with third-party integration, several service provider protocols are used to facilitate the on-boarding of new dApps, services, tools, and more.

Reviewer: ```suggestion
The EON ecosystem is perpetually changing with new partners and integrations. To accommodate the compatibility of EON platform with third-party integration, several service provider protocols are used to facilitate the onboarding of new dApps, services, tools, and more.
```

See the [EON Ecosystem](https://eon.horizen.io/ecosystem) for a description of current service providers.    

# Service Providers: Technology Features

> As new partners and integrations are on-boarding to the EON ecosystem, the need to use the following service providers’ technologies is imperative for a successful EVM-compatible sidechain:

Reviewer: ```suggestion
As new partners and integrations are onboarding to the EON ecosystem, the need to use the following service providers’ technologies is imperative for a successful EVM-compatible sidechain:
```

## Ankr  

Ankr provides a Web3 infrastructure that has many use cases, from Web3 API, staking, blockchain deployment tool, and smart contract execution. 

> Request for [EON Ankr-specific](http://info@horizen.io) documentation. 

Reviewer 2: I think we are trying to make this an email link. If so, it should be like the following:

```suggestion
Request for [EON Ankr-specific](mailto:info@horizen.io) documentation. 
```

The Band Protocol provides reliable, secure, and real-time data to smart contracts on EON and other blockchain networks.

See the [Band Protocol](https://docs.bandchain.org/develop/supported-blockchains/) documentation for more information.

## Covalent

Covalent is a hosted Web3 data provider offering a Unified API to bring full transparency and visibility to assets across all blockchain networks.

Developers use Covalent to build multi-chain applications like crypto wallets, NFT galleries, and investor dashboard tools utilizing data from various blockchain networks. 

See the [Covalent](https://www.covalenthq.com/docs/networks/horizen/) documentation for more information. 

## Flair

Flair provides real-time and historical custom data indexing for EON. It offers reusable indexing primitives (such as fault-tolerant RPC ingestors, custom processors, re-org aware database integrations) to make it easy to receive, transform, store and access your on-chain data.

> See the [Flair](https://github.com/HorizenOfficial/evm-documentation/pull/78) documentation for more information. 

Author: yes, this is the correct jump/link

## The Graph Protocol

The Graph is an indexing protocol for organizing blockchain data and accessible with GraphQL.

> Request for [EON Graph-specific](http://info@horizen.io) documentation.

Reviewer 2: Are we trying to achieve this?

```suggestion
Request for [EON Graph-specific](mailto:info@horizen.io) documentation.
```

## Pyth Protocol

The Pyth protocol allows market participants to publish pricing information on-chain for others to use. The Pyth Network is an oracle that publishes financial market data to multiple blockchains. 

See the [Pyth](https://docs.pyth.network/documentation/pythnet-price-feeds/evm) documentation for more information.

## Relink Protocol/Chainlink VRF Proxy

Relink has developed a proxy service solution (Chainlink VRF Proxy) that forwards the random data request to an officially Chainlink supported chain, where it processes the result accordingly.

 

> A Relink oracle network signs the data generated by Chainlink and passes the original data along with the signatures to the callback method. Signature checks in the consumer base contract ensure that the data cannot be altered in transit from the relink backend.

Reviewer: ```suggestion
A Relink oracle network signs the data generated by Chainlink and passes the original data along with the signatures to the callback method. Signature checks in the consumer base contract ensure that the data cannot be altered in transit from the Relink backend.
```
